### ISSUE TITLE
hotspot text always visible

### ISSUE THREAD

**Issue by almarass1** [mpetroff/pannellum#1160]
Hi,
custom hotspots have the possibility to make hotspot text always visible by setting the parameter visibility: visible.
Is it possible to make hotspots text always visible also in the case of ordinary hotspots both of type info and scene?

Thanks,
Alex

**Comment by almarass1** [comment]
As suggested by https://github.com/mpetroff/pannellum/issues/578, adding div.pnlm-tooltip span {visibility: visible !important;} seems to work

**Comment by almarass1** [comment]
Is it possible to change hotspot texts size and font?

Thanks,
Alex

**Comment by mpetroff** [comment]
> Is it possible to change hotspot texts size and font?

Yes, just set additional CSS properties for `div.pnlm-tooltip span`. For more extensive changes, the [custom hot spot example](https://pannellum.org/documentation/examples/custom-hot-spots/) can serve as a starting point.
